conf.py

```diff
@@ -79,6 +79,8 @@
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
+#Properly format dashes
+smartquotes=True
```

Comment:
This commit changes the formatting but I don't quite see it as a pure "fix". This turns on a feature (that I, personally loathe as it harms plaintext copy-paste) to change how dashes leading dashes are formatted. Was this a problem when updating other things and building locally? In order to review this change alone I think it would be best to split it and all of the reformating it touches out from the other changes.

Reply:
It was an issue in the check build for the PR below (not just a local issue). In the .rst files `\\--` was originally displaying  as `--`, but after adding the mandatory .readthedocs.yaml and subsequent updates, it began appearing as `\--`. Maybe smartquotes isn't necessary and there's a better fix. I can split these changes out and avoid touching the formatting, but the double dashes are going to look odd in the interrim.

Reply:
Looks like smartquotes was unnecessary after all. I've removed it. Dropping from `\\--` to `\--` seems to be enough.

Reply:
Got it. Thanks for following through. I'm torn about whether or not to split this into a separate commit so it can be added to an [ignore-revs-file](https://git-scm.com/docs/git-blame#Documentation/git-blame.txt---ignore-revs-fileltfilegt) when reviewing file changes. @cottsay, on an A Little to A Lot scale how much do you care?

I'm willing do the rebase, split, and merge myself if your answer is >= "Not even at all".

Reply:
> how much do you care?

Indifferent, I'll defer to your preference here.